heron/ui/src/python/main.py

```diff
@@ -134,14 +135,21 @@ def main():
 
   # log additional information
   command_line_args = vars(parsed_args)
-  address = socket.gethostbyname(socket.gethostname())
+
+  address = '0.0.0.0'
+  if command_line_args['address']:
+    address = command_line_args['address']
+  else:
+    Log.warn("Bindings wildcard by default")
   Log.info("Listening at http://%s:%d", address, command_line_args['port'])
   Log.info("Using tracker url: %s", command_line_args['tracker_url'])
 
   # pass the options to tornado and start the ui server
-  define_options(command_line_args['port'], command_line_args['tracker_url'])
+  define_options(command_line_args['address'],
+                 command_line_args['port'],
+                 command_line_args['tracker_url'])
   http_server = tornado.httpserver.HTTPServer(Application())
-  http_server.listen(command_line_args['port'])
+  http_server.listen(command_line_args['port'], address=command_line_args['address'])
```

Comment:
my bad! I did not read create_parsers() carefully. The default arguments have been set up. Should I add tracker address in this pull request?